javacutil/src/org/checkerframework/javacutil/AnnotationBuilder.java

```diff
@@ -180,6 +180,30 @@ public void copyElementValuesFromAnnotation(
         return;
     }
 
+    /**
+     * Copies the specified element values from the given annotation, using the specified name
+     * mapping. If an element specified as a target in the mapping doesn't exist in the annotation
+     * to be built, an error is raised. If an element from the given annotation is not in the
```

Comment:
Rather than saying "an error is raised", I would state the requirement:  each value in the mapping must be an element name in the annotation being built.  (But isn't that obvious?)